packages/common/src/utils/slugs.ts

```diff
@@ -53,17 +53,13 @@ export const getParentSlug = (slug: string) =>
  */
 export const getLabelFromSlug = (slug: string) => slug.split('/').pop() ?? slug;
 
-/**
- * Get the ltree path from a slug with hierarchy
- * For example, "parent-space/child-space" will return "parent_space.child_space"
- * Verifies that each slug is a valid PostgreSQL ltree label (A-Za-z0-9_)
- * @param slug - The slug to get the ltree path from
- * @returns The ltree path
- */
-// TODO: needs backwards compat
+// /parent/child -> parent.child
 export const getLtreePathFromSlug = (slug: string) => {
```

Comment:
inverse of this would require only replacing `___` in the middle because there's a chance of having this:

```
getLtreePathFromSlug(`-parent-/-child-`) // old broken slugs
=> "_parent_____child_"
```